vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsMessageItem.kt

```diff
@@ -105,6 +106,9 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : AbsBaseMessageItem<H>
         } else {
             holder.timeView.isVisible = false
         }
+
+        holder.additionalTopSpace.isGone = !attributes.informationData.messageLayout.addTopMargin
```

Comment:
Nit: it should be better to avoid using a negation here. So I would prefer to see
 
```kotlin
holder.additionalTopSpace.isVisible = attributes.informationData.messageLayout.addTopMargin
```

`isGone` -> `isVisible` and remove the `!`

Anyway, this is not a blocker.